modules/identity-provider-secret.adoc

```diff
@@ -17,15 +21,33 @@ namespace to contain the client secret, client certificates, and keys.
 * You can define an {product-title} `Secret` object containing a string
 by using the following command.
 +
+====
 [source,terminal]
 ----
-$ oc create secret generic <secret_name> --from-literal=clientSecret=<secret> -n openshift-config
+ifndef::basic[]
+$ oc create secret generic <secret_name> --from-literal=clientSecret=<secret> -n openshift-config[]
+endif::basic[]
+ifdef::basic[]
+$ oc create secret generic <secret_name> --from-literal=tls.crt=<secret> --from-literal=tls.key=<secret> -n openshift-config[]
```

Comment:
> -n openshift-config[]

You could self review :) to have found "`[]`" is added in line 28 and 31, please remove it. Line 43 and 46 has same issue.

Reply:
Actually, if creating a secret which uses key names `tls.crt` and `tls.key`, instead of `oc create secret generic`, we should prefer `oc create secret tls` as shown in the doc in https://github.com/openshift/openshift-docs/pull/43279#discussion_r828993659 .

Reply:
Changes have been made. :)